main_test.py

```diff
@@ -5,58 +5,226 @@
 
 from PIL import Image
 
-from main import app
+from app import app, init_app
+
+TEST_SCHEDULES = {
+    "aaardvark": {

```

Comment:
Rather than keeping the entire test JSON file written out in `main_test`, it's probably cleaner to load a `mock_data.json` file.